prisma/seed.mjs

```diff
@@ -67,8 +56,8 @@ function randomSpecialist({ districts, specializations, therapies }) {
     yearsOfExperience: faker.number.int({ min: 1, max: 30 }),
     // take one of these
     formatOfWork: faker.helpers.arrayElement(['BOTH', 'ONLINE', 'OFFLINE']),
-    placesOfWork: {
-      create: randomPlacesOfWork,
+    addresses: {
+      create: randomAddresses,
```

Comment:
Shouldn't "randomAddresses" function be called?

Reply:
randomAddresses is an array of addresses that gets generated right above in this function

Reply:
Oh I see. Ok then